rustfmt.toml

```diff
@@ -0,0 +1,5 @@
+group_imports = "StdExternalCrate"
```

Comment:
It would be nice if the rustfmt changes were in another commit. If it is too difficult to separate the two, could you add something to the commit message about the change to the rustfmt config.

Reply:
Ok, I'll split them up.